src/main/java/space/obminyashka/items_exchange/service/MailService.java

```diff
@@ -15,4 +18,12 @@ public interface MailService {
      * @throws IOException when service is unavailable or some unexpected case happened
      */
     void sendMail(String emailTo, EmailType emailType, Locale locale) throws IOException;
+
+    /**
+     * Validate code and if it is existed and unexpired, users`s email will be confirmed
```

Comment:
**Validate users' email by previously generated UUID code** would describe the method a bit better.